src/scope-hoisting/hoist.js

```diff
@@ -296,9 +297,11 @@ module.exports = {
           p.isSequenceExpression()
       );
       if (!parent.isProgram() || bail) {
-        asset.dependencies.get(args[0].value).shouldWrap = true;
+        asset.dependencies.get(source).shouldWrap = true;
       }
 
+      asset.cacheData.imports['$require$' + source] = [source, '*'];
```

Comment:
This could potentially conflict with a real import, right? e.g. `import $require$foo from './foo'`

Reply:
hmm, in imports we would use `$id$import$$require$foo` as key :
```js
asset.cacheData.import = {
  "$require$foo": ['foo', '*'],
  "$id$import$$require$foo": ['./foo', 'default']
}
```

or is there another place where it would conflict?

Reply:
oh right...